bin/create-k8s-config-maps-and-secrets

```diff
@@ -18,6 +18,8 @@ HARMONY_VERSION=$(grep version ./package.json \
 USER_AGENT="harmony/${HARMONY_VERSION} (${CLIENT_ID})"
 
 SERVICES_YML=$(< config/services.yml)
+# needed for k8s services running locally that access the dB
+DATABASE_URL=${DATABASE_URL/localhost/postgres}
```

Comment:
This is a string substitution, so as long as "localhost" does not appear in the DATABASE_URL it will not be affected. I don't think it would for any deployed environments.